### DISCUSSION TITLE
Release v0.27

### DISCUSSION THREAD

**Discussion by Coool** [NewPipeX/NewPipeX#32] [Questions] [answered]
Hi!

Are you planning release 0.27 version? NewPipe updated!

Thanks.

**Comment by DI555** [comment]


**Reply by Coool** [reply]
Are you aware of [BraveNewPipe](https://github.com/bravenewpipe/NewPipe), which supports Bitchute & Rumble?

**Comment by javulticat** [comment, marked as answer]
[`v0.27.0-1` of NewPipe X is now available](https://github.com/NewPipeX/NewPipeX/releases/tag/v0.27.0-1)
